dev/fattest.simplicity/src/componenttest/rules/repeater/EE8FeatureReplacementAction.java

```diff
@@ -52,7 +52,11 @@
                                                  "mdb-3.2",
                                                  "componenttest-1.0",
                                                  "txtest-1.0",
-                                                 "websocket-1.1" };
+                                                 "websocket-1.1",
```

Comment:
Same changes are needed for EE7FeatureReplacementAction

Reply:
Should be "jms-2.0", not 3.0

Reply:
Changes made.